css/index.css

```diff
@@ -54,29 +54,58 @@ table {
 html, body {
     height: 100%;
     font-family: 'Roboto', sans-serif;
+    font-size: 62.5%;
 }
 
 h1, h2, h3, h4, h5 {
-    font-size: 18px;
+    font-size: 1.8rem;
     margin-bottom: 15px;
     font-family: 'Rubik', sans-serif;
 }
 
 p {
     line-height: 1.4;
+    font-size: 1.5rem;
 }
 
 .container {
-    width: 800px;
+    width: 100%;
+    max-width: 1000px;
```

Comment:
We talked about this previously, why max-width is 800px and the rest is margins. We end up having the paragraphs in `.top-content` breach the wanted 800px.
![breach](https://user-images.githubusercontent.com/12449668/37355677-877c2bb0-26a1-11e8-9f06-a2c6028fd80b.png)